.release-notes/mode-os.md

```diff
@@ -0,0 +1,5 @@
+## Make FileMode._os public
+
+FileMode has always had a private method for getting an integer representation of the file mode. However, it was private and only available for use within the `files` package.
+
+It is now public as `FileMode.os`.
```

Comment:
`os` is kind of a weird name for this - as a private method that didn't matter much, but now that it's public, it kind of does. I'd kind of expect `os` to return me something that tells what operating system the filesystem is running within, as little sense as that may make.

I might have preferred something else, though my night-weary brain is unsure what the right thing to call it would be. Maybe just `u32`?